### ISSUE TITLE
flake8 config not recognized

### ISSUE THREAD

**Issue by svartkanin** [mfussenegger/nvim-lint#600]
I'm probably doing something wrong but in my current setup the `.flake8` configuration file doesn't seem to be used when the linter is applied? 

Example `test.py` which includes tabs
```
def test():
	print('hello')
```

`.flake8` config file
```
[flake8]
ignore = W191
show-source = True
```

linter configuration 
`nvim/lua/plugins/nvim-lint.lua`
```
return {
  'mfussenegger/nvim-lint',
  event = { 'BufReadPre', 'BufNewFile' },
}
```

`nvim/after/plugins/nvim-lint.lua`
```
require('lint').linters_by_ft = {
  markdown = { 'vale' },
  python = { 'mypy', 'flake8' },
}

local lint = require 'lint'
local lint_augroup = vim.api.nvim_create_augroup('lint', { clear = true })

vim.api.nvim_create_autocmd({
  'BufEnter', -- trigger whenever we open a new buffer or we move the cursor into another buffer
  'BufWritePost', -- trigger whenever we save a buffer
  'InsertLeave', -- trigger whenever we leave insert mode
}, {
  group = lint_augroup,
  callback = function() -- will trigger when any of the above events are triggered
    lint.try_lint() -- the lint plugin which we loaded will try to execute linting
  end,
})
```

Which will show a flake8 warning 
![image](https://github.com/mfussenegger/nvim-lint/assets/19396770/060def13-29df-4738-9535-f772e72920f8)

However, running `flake8` manually with 
```$ flake8 test.py```

will not show the warning as it locates the `.flake8` configuration inside the folder

**Comment by svartkanin** [comment]
Thanks for the confirmation. I found the culprit, had this in my lsp config 
```
    require('mason-lspconfig').setup {
      handlers = {
        function(server_name)
          local server = servers[server_name] or {}
          -- This handles overriding only values explicitly passed
          -- by the server configuration above. Useful when disabling
          -- certain features of an LSP (for example, turning off formatting for tsserver)
          server.capabilities = vim.tbl_deep_extend('force', {}, capabilities, server.capabilities or {})
          require('lspconfig')[server_name].setup(server)
        end,
      },
    }
```
which seems to have disabled the loading of the flake8 config.
